ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java

```diff
@@ -55,7 +54,7 @@
   public static long GENESIS_SLOT = 4294967296L; // 2^32
```

Comment:
We could potentially change `GENESIS_SLOT = 0` , create `GENESIS_OFFSET = 4294967296L` and in the serialize()/ deserialize methods do `UINT64 slot64 = this.slot + GENESIS_OFFSET` and `this.slot= slot64 - GENESIS_OFFSET` respectively

Reply:
It's likely that `GENESIS_SLOT` will end up being zero in any case. See comments on https://github.com/ethereum/eth2.0-specs/pull/843 for evidence.

Reply:
Since changing `GENESIS_SLOT` to 0 and introducing a new variable `GENESIS_OFFSET` touches a lot of different parts of the code, I just set `GENESIS_SLOT = 0`. If this change ends up not taking place, we can update the code later on, but for simplicity sake, decided it was best to solve it as such.